CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
```

Comment:
I prefer this living in the `README.md` or `CONTRIBUTING.md`. It keeps the `CHANGELOG.md` concise and to the point. 

Let's go with `CONTRIBUTING.md`